pypit/arutils.py

```diff
@@ -1076,6 +1076,61 @@ def subsample(frame):
     indices = coordinates.astype('i')
     return frame[tuple(indices)]
 
+def trace_gweight(fimage, xcen, ycen, sigma, invvar=None):
+    """ Port of SDSS trace_gweight algorithm
+    Parameters
+    ----------
+    fimage
+    xcen
+    ycen
+    sigma
+    invvar
+
+    Returns
+    -------
+    xnew
+    xerr
+
+    """
+    # import
+    from scipy.special import erf
+    # Setup
+    nx = fimage.shape[1]
+    xnew = xcen.copy()
+    xerr = 0.0 * xnew # Allocate memory
```

Comment:
Could this be replaced with: `xerr = np.zeros_like(xnew)` or `np.zeros(xnew.shape, dtype=np.float)`?